Survey/Survey/Models/SurveyViewModel.swift

```diff
@@ -0,0 +1,27 @@
+import Foundation
+import Combine
+
+class SurveyViewModel: ObservableObject {
+    @Published var inquiries: [Inquiry] = []
+    private var cancellables = Set<AnyCancellable>()
+    private let mockServer = MockServer()
+
+    func loadInquiries() {
+        mockServer.loadInquiries()
+            .receive(on: DispatchQueue.main)
+            .sink(receiveCompletion: { completion in
+                switch completion {
+                case .finished:
+                    break
+                case .failure(let error):
+                    print("Failed to load inquiries: \(error)")
+                DispatchQueue.main.asyncAfter(deadline: .now() + 600) {
```

Comment:
i'm not sure permanent retry every 600ms is the right mechanism here. let's discuss

Reply:
It is means every 10min as in acceptance criteria

Reply:
ah okay yeah you're right. I'm so used to these being in ms. :)